Gemfile.lock

```diff
@@ -308,8 +309,5 @@ DEPENDENCIES
   unicorn (= 4.8.3)
   webmock (= 1.20.4)
 
-RUBY VERSION
```

Comment:
not sure - I just ran `bundle install` - I can revert if you think it's important? (I'm not really sure what this does?)